batch/batches/Convert/KAsyncConvert.class.php

```diff
@@ -295,8 +295,16 @@ private function moveFile(KalturaBatchJob $job, KalturaConvertJobData $data)
 					$fileSize=KBatchBase::foldersize($data->destFileSyncLocalPath);
 				else
 					$fileSize = kFile::fileSize($data->destFileSyncLocalPath);
-			
-				kFile::moveFile($data->destFileSyncLocalPath, $sharedFile);
+
+				if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN')
```

Comment:
Instead of checking if its windows why not add a config on the worker config to force it to go via this flow. This way we can re use it the future if needed without being dependent on if its windows or not.